src/utils.js

```diff
@@ -85,7 +85,7 @@ const checkExistAndDecryptSigningKey = ({ pw }, signingKeyFile) =>
     const signingKey = JSON.parse(rawSigningKey);
 
     decryptValue(signingKey.private_key_cipher_text, pw)
-      .then(res)
+      .then(singingPrivateKey => res({ singingPrivateKey, signingKey }))
```

Comment:
why did you change this? For me it makes no sense to resolve with the used singing key.